src/System.Drawing.Common/tests/SystemBrushesTests.cs

```diff
@@ -0,0 +1,77 @@
+            yield return Brush(() => SystemBrushes.Desktop, SystemColors.Desktop);
+            yield return Brush(() => SystemBrushes.GradientActiveCaption, SystemColors.GradientActiveCaption);
+            yield return Brush(() => SystemBrushes.GradientInactiveCaption, SystemColors.GradientInactiveCaption);
+            yield return Brush(() => SystemBrushes.GrayText, SystemColors.GrayText);
+            yield return Brush(() => SystemBrushes.Highlight, SystemColors.Highlight);
+            yield return Brush(() => SystemBrushes.HighlightText, SystemColors.HighlightText);
+            yield return Brush(() => SystemBrushes.HotTrack, SystemColors.HotTrack);
+            yield return Brush(() => SystemBrushes.InactiveBorder, SystemColors.InactiveBorder);
+            yield return Brush(() => SystemBrushes.InactiveCaption, SystemColors.InactiveCaption);
+            yield return Brush(() => SystemBrushes.InactiveCaptionText, SystemColors.InactiveCaptionText);
+            yield return Brush(() => SystemBrushes.Info, SystemColors.Info);
+            yield return Brush(() => SystemBrushes.InfoText, SystemColors.InfoText);
+            yield return Brush(() => SystemBrushes.Menu, SystemColors.Menu);
+            yield return Brush(() => SystemBrushes.MenuBar, SystemColors.MenuBar);
+            yield return Brush(() => SystemBrushes.MenuHighlight, SystemColors.MenuHighlight);
+            yield return Brush(() => SystemBrushes.MenuText, SystemColors.MenuText);
+            yield return Brush(() => SystemBrushes.ScrollBar, SystemColors.ScrollBar);
+            yield return Brush(() => SystemBrushes.Window, SystemColors.Window);
+            yield return Brush(() => SystemBrushes.WindowFrame, SystemColors.WindowFrame);
+            yield return Brush(() => SystemBrushes.WindowText, SystemColors.WindowText);
+        }
+
+        public static object[] Brush(Func<Brush> brushThunk, Color expectedColor) => new object[] { brushThunk, expectedColor };
+
+        [ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        [MemberData(nameof(SystemBrushes_TestData))]
+        public void SystemBrushes_Get_ReturnsExpected(Func<Brush> brushThunk, Color expectedColor)
+        {
+            SolidBrush brush = Assert.IsType<SolidBrush>(brushThunk());
+            Assert.Equal(expectedColor, brush.Color);
+            Assert.Throws<ArgumentException>(null, () => brush.Color = Color.Red);
+
+            Assert.Same(brush, brushThunk());
+        }
+
+        [Fact]
+        public void FromSystemColor_NotSystemColor_ThrowsArgumentException()
+        {
+            Assert.Throws<ArgumentException>(null, () => SystemBrushes.FromSystemColor(Color.Blue));
+        }
+
+        [Fact]
+        public void SystemBrushes_PrivateCtor_Success()
+        {
+            ConstructorInfo constructor = typeof(SystemBrushes).GetConstructor(BindingFlags.NonPublic | BindingFlags.Instance, null, new Type[0], null);
```

Comment:
I made a similar comment on the other PR: I'm not sure we need a test like this. The private constructor might be needed for things like serialization, though; not sure. In any case I don't think we want a test case exactly like this.